src/views/Orders.vue

```diff
@@ -220,7 +224,9 @@ export default defineComponent({
       currentJobStatus: '',
       freqType: '',
       isJobDetailAnimationCompleted: false,
-      isDesktop: isPlatform('desktop')
+      isDesktop: isPlatform('desktop'),
+      autoCancelDays: '',
+      updatedAutoCancelDays: ''
```

Comment:
I think we could have variable names as `autoCancelDays` and `currentAutoCancelDays`. Also, instead of checking in update method we should either disable the save button if both the values are same

Reply:
Made the requested change.